lib/src/http/base_request.dart

```diff
@@ -44,7 +44,8 @@ abstract class BaseRequest implements FluriMixin, RequestDispatchers {
   /// Information about this request related to retries is also available here.
   /// This includes the current number of attempts and the current list of
   /// failures.
-  RequestAutoRetry? autoRetry;
+  RequestAutoRetry get autoRetry;
+  set autoRetry(RequestAutoRetry value);
```

Comment:
Is this any different than
```suggestion
  RequestAutoRetry autoRetry;
```